client/src/components/form/Form.tsx

```diff
@@ -445,6 +445,68 @@ export function FormDocumentInformation({
     setPageRanges: React.Dispatch<React.SetStateAction<PageRange[] | undefined>>;
 }) {
 
+    const [aggregatedStakeholders, setAggregatedStakeholders] = useState<string[]>([]);
+    const [aggregatedScales, setAggregatedScales] = useState<string[]>([]);
+    const [aggregatedTypes, setAggregatedTypes] = useState<string[]>([]);
+    const [newStakeholder, setNewStakeholder] = useState('');
+    const [newType, setNewType] = useState('');
+    const [newScale, setNewScale] = useState('');
+
+    useEffect(() => {
+        const fetchAggregateData = async () => {
+            try {
+                const aggregateData = await API.getAggregatedData();
+                if (aggregateData) {
+                    setAggregatedStakeholders(aggregateData.stakeholders);
+                    setAggregatedScales(aggregateData.scales.map(scale => scale.toLocaleString()));
+                    setAggregatedTypes(aggregateData.types);
+                }
+
+            } catch (error: any) {
+                console.error(error);
+            }
+        };
+        fetchAggregateData();
+    }, []);
+
+    const handleAddNewStakeholder = (e: React.FormEvent) => {
+        e.preventDefault();
+        if (newStakeholder && !aggregatedStakeholders.includes(newStakeholder)) {
+            const updatedStakeholders = [...aggregatedStakeholders, newStakeholder];
+            setAggregatedStakeholders(updatedStakeholders);
+            setStakeholders([...stakeholders, newStakeholder]);
+        }
+        setNewStakeholder('');
+    };
+
+    const handleAddNewType = (e: React.FormEvent) => {
+        e.preventDefault();
+        if (newType && !aggregatedTypes.includes(newType)) {
+            const updatedTypes = [...aggregatedTypes, newType];
+            setAggregatedTypes(updatedTypes);
+            setType(newType);
+        }
+        setNewType('');
+    };
+
```

Comment:
<img width="1429" alt="Screen Shot 2024-12-18 at 20 18 20" src="https://github.com/user-attachments/assets/ab223416-9402-4218-a134-0555378bdfc6" />
<img width="1432" alt="Screen Shot 2024-12-18 at 20 19 11" src="https://github.com/user-attachments/assets/4b4a5838-aa43-4766-b0f2-c64af5b0fc50" />
<img width="1426" alt="Screen Shot 2024-12-18 at 20 19 36" src="https://github.com/user-attachments/assets/9eb6c08b-45f5-43a3-b2ce-2fbe344cca9d" />


there are these three errors on the page. However, they are not preventing the submission.

Reply:
validateDomNesting and File upload were already on Dev. I couldn't figure out what causes the child error with the same key